controllers/dockercluster_controller.go

```diff
@@ -38,88 +46,131 @@ type DockerClusterReconciler struct {
 // +kubebuilder:rbac:groups=infrastructure.cluster.x-k8s.io,resources=dockerclusters,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=infrastructure.cluster.x-k8s.io,resources=dockerclusters/status,verbs=get;update;patch
 
-// Reconcile handles DockerCluster events
-func (r *DockerClusterReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, rerr error) {
+// Reconcile reads that state of the cluster for a DockerCluster object and makes changes based on the state read
+// and what is in the DockerCluster.Spec
+func (r *DockerClusterReconciler) Reconcile(req reconcile.Request) (_ reconcile.Result, reterr error) {
```

Comment:
This should be `ctrl.Result`? not a `reconcile.Result` unless they are the same thing.

Reply:
They are the same thing, but I'm switching to `ctrl` (same to Kubebuilder 2)